scripts/dataset_tag_editor/dataset_tag_editor.py

```diff
@@ -154,7 +154,7 @@ def sort_tags(self, tags: List[str], sort_by: str, sort_order: str):
                 return sorted(tags, key=lambda t:(len(t), t), reverse=False)
             elif sort_order == 'Descending':
                 return sorted(tags, key=lambda t:(-len(t), t), reverse=False)
-        return []
+        return tags
```

Comment:
Returning an empty array for invalid sorting options seemed very wrong to me, so I changed the fallback case to instead return the original array as-is.

I'm not entirely happy with this, since the most correct behavior would be to return an exact **copy** of the input array, but this seemed unnecessarily complex for what's, more-or-less, a dead code pathway.